Packs/Campaign/Scripts/GetCampaignIndicatorsByIncidentId/GetCampaignIndicatorsByIncidentId.py

```diff
@@ -30,13 +26,16 @@ def get_indicatos_from_incidents(incident_ids: List) -> List:
     Returns:
         List of the campaign indicators.
     """
-    indicators_query = 'investigationIDs:({})'.format(' '.join('"{}"'.format(id_) for id_ in incident_ids))
+    indicators_query = f"""investigationIDs:({' '.join(f'"{id_}"' for id_ in incident_ids)})"""
     fields = ['id', 'indicator_type', 'investigationIDs', 'relatedIncCount', 'score', 'value']
     indicators_args = {'query': indicators_query, 'limit': '150', 'populateFields': ','.join(fields)}
-    return execute_command('GetIndicatorsByQuery', args=indicators_args)
+    res = execute_command('GetIndicatorsByQuery', args=indicators_args)
+    if is_error(res):
+        return_error(f'Error in GetIndicatorsByQuery. {get_error(res)}')
```

Comment:
does `return_error` stop execution? Or do we get to `return res` in case of an error anyway?

Reply:
return_error docstring:

```Returns error entry with given message and exits the script```